src/components/media_manager/src/socket_streamer_adapter.cc

```diff
@@ -70,6 +70,8 @@ SocketStreamerAdapter::SocketStreamer::SocketStreamer(
     ip_(ip),
     port_(port),
     header_(header),
+    server_socket_(),
```

Comment:
@malirod, please provide a proof-link
I get the idea of the "best practice"  but still don't see advantages.